eurekaclient/servicelookup.json

```diff
@@ -0,0 +1 @@
+[{"uri": "/book", "app": "BOOK", "secure": false},{"uri":"/httpbin", "app": "BOOK", "secure": false}]
```

Comment:
Is this something required for the client, or is it just something that got let in accidentally?